share/governance.md

```diff
@@ -0,0 +1,111 @@
+
+* Advocating for use of the project
+* Informing developers of project strengths and weaknesses from a user’s perspective
+* Providing moral support (a ‘thank you’ goes a long way)
+* Writing documentation and tutorials
+* Filing bug reports and feature requests
+* Participating on the discussion board
+
+
+Users who continue to engage with the project and its community will often find themselves becoming more and more involved. Such users may then go on to become contributors, as described below.
+
+*How to become one*: Use a Vox Pupuli project
+
+
+Collaborators
+-------------
+
+
+Collaborators are community members who submit patches to the project. These patches may be a one-time occurrence or occur over time. It is anticipated that collaborators' contributions may be small at first, growing more considerable once the collaborator has built up the necessary knowledge, experience, and confidence.
+
+*How to become one*: Submit a pull request to a Vox Pupuli project
+
+
+Contributors
+------------
+
+
+Contributors are collaborators who have shown wide dedication to the Vox Pupuli project in general or deep dedication to one project in particular, and the ability to work well with collaborators, users, and other contributors. The contributors have responsibilities beyond the collaborators. In particular, contributors formally decide on whether a pull request is merged or a release is cut. Contributors are recognized and valued for their contributions, in whatever form those contributions take. Tasks such as communication, documentation, art, packaging, and testing are all valued contributions. A contributor will use lazy consensus to decide on whether to merge a pull request from a collaborator. If the discussion is no longer moving towards a consensus, the PMC must vote via lazy consensus on whether the patch should be applied.
+
+*How to become one*: Be a collaborator and be nominated to the PMC as a contributor. Nominations should be sent to <email address for nominations>. You may nominate yourself. It is also common to give contributor status to an individual who donates code to the project by migrating a repository to the github namespace.
+
+
+Project Management Committee (PMC)
+----------------------------------
+
+
+The project management committee has responsibilities beyond contributors that include participating in strategic planning, release planning, and approving changes to the governance model. They also have the duty of upholding the community code of conduct(https://voxpupuli.org/coc/) and ensuring its values. The PMC is also responsible for updating the community code of conduct. It also makes decisions when community consensus cannot be reached. 
+
+The PMC has final say over who can become a committer and will use lazy consensus for approval. Discussion over committer nominations will be done in private.
```

Comment:
> The PMC has final say over who can become a committer

&

> Discussion over committer nominations will be done in private.

thus far we've been doing it like so:

```irc
< alice> how come $person doesn't have merge access yet??
< bob> i dunno, ask if they wanna join
```

i like this model, and i wish it could be encoded in a form that's agile *and* bias-free.